### ISSUE TITLE
My Site crashes (Google Chrome says: "Aw, Snap!") with I18NConf

### ISSUE THREAD

**Issue by obiwarn** [yoolab/iron-router-i18n#48]
Whenever I remove the languages  from Router config and place them in my I18NConf. My Site crashes. No errors are displayed.

My config:

Router.configure
    i18n:
        serverSide: true
        defaultLanguage: 'de'
        languages: ['en', 'de', 'fr']
        setLanguage: (lang) ->
            Language.setLanguage(lang)
        exclude:
            admin_paths: '^\/admin',
        server:
            exclude:
                sitemap: '^\/sitemap\.xml'

**Comment by martinopic** [comment]
What is "Language.setLanguage(lang)" actually doing? Can you try to remove the ```setLanguage``` configuration and to put it in a

```javascript
I18NConf.onLanguageChange(function(oldLang, newLang) {
  Language.setLanguage(newLang);
});

```


 The reported error seems to come from a infinite loop as if the two setLanguage methods are calling each other repeatedly (there is a mechanism preventing this in Iron Router i18n setLanguage method).

**Comment by martinopic** [comment]
Also, this could be related to #50 fixed in 0.5.6 version just released today.

**Comment by martinopic** [comment]
Haven't received any further feedback on this for two weeks, closing.
